yellowbrick/text/postag.py

```diff
@@ -232,10 +230,14 @@ def _handle_universal(self, X,y):
                     for _, tag in tagged_sent:
                         if tag == "SPACE":
                             continue
-                        counter = self.pos_tag_counts_[y[idx]] if self.stack else self.pos_tag_counts_
+                        if self.stack:
+                            counter = self.pos_tag_counts_[y[idx]] 
+                        else: 
+                            counter = self.pos_tag_counts_['documents']
+                        
                         counter[jump.get(tag, "other")] += 1
 
-    def _handle_treebank(self, X,y):
+    def _handle_treebank(self, X, y=None):
```

Comment:
Thank you for making these updates!

Reply:
It's not very important here - but let's please be sure that we are also updating the docstrings with all the parameters. Since this is library code, all of our docstrings serve as documentation that users might see with `help()` or built in the docs.